lib/promscrape/client_test.go

```diff
@@ -0,0 +1,82 @@
+package promscrape
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/promauth"
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/proxy"
+	"github.com/stretchr/testify/require"
+)
+
+func TestReadData_HTTPConnectPassthrough(t *testing.T) {
+	ctx := context.Background()
+	trequire := require.New(t)
```

Comment:
Could you please remove `testify` dependency?
We're usually writing tests with standard Go functions and just making comparisons manually, you can refer to these tests for example: https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/lib/promscrape/config_test.go

Reply:
done!